ReactWindows/js/Components/Picker/PickerIOS.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule PickerIOS
+ */
+'use strict';
```

Comment:
See [https://github.com/facebook/react-native/blob/master/Libraries/Picker/PickerIOS.android.js.](https://github.com/facebook/react-native/blob/master/Libraries/Picker/PickerIOS.android.js.)

You'll want to name this file PickerIOS.windows.js